src/test/java/com/g4s8/ghman/data/PgUserITCase.java

```diff
@@ -16,32 +16,123 @@
  */
 package com.g4s8.ghman.data;
 
+import com.g4s8.ghman.user.GhAuthException;
+import com.g4s8.ghman.user.User;
+import com.g4s8.hamcrest.json.JsonHas;
+import com.g4s8.hamcrest.json.JsonValueIs;
+import com.jcabi.github.RtGithub;
 import java.io.IOException;
+import org.cactoos.iterable.IterableOf;
+import org.cactoos.text.FormattedText;
+import org.hamcrest.core.AllOf;
+import org.hamcrest.core.IsEqual;
 import org.junit.jupiter.api.Test;
 import org.junit.jupiter.api.extension.ExtendWith;
 import org.llorllale.cactoos.matchers.Assertion;
 import org.llorllale.cactoos.matchers.Throws;
+import org.mockito.Mockito;
+import org.telegram.telegrambots.api.objects.Chat;
 
 /**
  * Test for {@link com.g4s8.ghman.data.PgUser}.
  * @since 1.0
- * @todo #7:30min Continue implement this integration test: add cases for all
- *  methods of {@link com.g4s8.ghman.data.PgUser} class. Do not forget about
- *  all possible exceptions.
+ * @checkstyle ClassDataAbstractionCouplingCheck (500 lines)
  */
 @ExtendWith(DatabaseExtension.class)
 final class PgUserITCase {
 
     @Test
     void throwsExceptionIfUserIsNotFound() {
         new Assertion<>(
-            "Unexpected exception while selecting not existing user",
-            () -> new PgUser(DatabaseExtension.dataSource(), 0).github(),
+            "Should throw exception while selecting not existing user",
+            () -> new PgUser(DatabaseExtension.dataSource(), -1).github(),
             new Throws<>(
                 "Failed to select token",
                 IOException.class
             )
         ).affirm();
     }
 
+    @Test
+    void returnsUserId() {
+        final long uid = 45;
+        new Assertion<>(
+            "Should return user id",
+            new PgUser(DatabaseExtension.dataSource(), uid).uid(),
+            new IsEqual<>(uid)
+        ).affirm();
+    }
+
+    @Test
+    void throwsExceptionIfTokenIsEmpty() throws IOException {
+        final User user = new PgUsers(DatabaseExtension.dataSource())
+            .user(getChat(458L, "test1"));
+        user.authorize("");
+        new Assertion<>(
+            "Should throw exception if token is empty",
+            user::github,
+            new Throws<>(
+                "Github token was not found",
+                GhAuthException.class
+            )
+        ).affirm();
+    }
+
+    @Test
+    void returnsGhUser() throws IOException {
+        final User user = new PgUsers(DatabaseExtension.dataSource())
+            .user(getChat(408L, "test2"));
+        final String token = "some_token";
+        user.authorize(token);
+        new Assertion<>(
+            new FormattedText(
+                "Should return github API instance with token %s", token
+            ).toString(),
```

Comment:
@olenagerasimova please use `asString()` here (and maybe it's not worh the complexification to build a proper string like this, by reading the test things already clear.